extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerProcessor.java

```diff
@@ -118,6 +118,9 @@ private void createContainerImage(ContainerImageConfig containerImageConfig, Doc
         log.infof("Pushed container image %s (%s)\n", image, reader.getImageId());
 
         if (pushRequested || containerImageConfig.push) {
+            if (!containerImageConfig.registry.isPresent()) {
+                log.info("No container image registry was set, so 'docker.io' will be used");
```

Comment:
```suggestion
                log.info("No container image registry was set, defaulting to 'docker.io'");
```

Reply:
I dunno if it's worth having an info for that.

Reply:
I don't agree with the proposed change, but I do think that a log message should be used. After all the registry is `Optional` so when it's not set the user should know that some default was used